code_schemes/s01e06.json

```
@@ -3,6 +3,118 @@
     "Name": "RQA S01E06",
     "Version": "0.0.0.1",
     "Codes": [
+        {
+            "CodeID": "code-ce93da6c",
+            "CodeType": "Normal",
+            "DisplayText": "yes, without reason",
```

```suggestion
            "DisplayText": "yes, no reason",
```
In the past, we've used that - No need to update here.

---

okay noted.